resources/views/torrent/peers.blade.php

```diff
@@ -83,29 +83,25 @@ class="{{ $p->user->group->icon }}" data-toggle="tooltip"
                                                 {{ $p->user->username }}</span></a>
                                 </td>
                             @endif
-                            @if ($p->seeder == 0)
-                                <td>
-                                    <div class="progress">
-                                        <div class="progress-bar progress-bar-striped active" role="progressbar"
-                                             aria-valuenow="{{ ($p->downloaded / $torrent->size) * 100 }}"
-                                             aria-valuemin="0"
-                                             aria-valuemax="100"
-                                             style="width: {{ ($p->downloaded / $torrent->size) * 100 }}%;">
-                                            {{ round(($p->downloaded / $torrent->size) * 100) }}%
-                                        </div>
-                                    </div>
-                                </td>
-                            @elseif ($p->seeder == 1)
-                                <td>
-                                    <div class="progress">
-                                        <div class="progress-bar progress-bar-striped active" role="progressbar"
-                                             aria-valuenow="100" aria-valuemin="0" aria-valuemax="100"
-                                             style="width: 100%;">
-                                            100%
-                                        </div>
+                            @php
```

Comment:
I would avoid using @php @endphp. I know it used in a few places in UNIT3D already but it is considered bad practice and they will be removed eventually so dont really want to add more.

Reply:
How about moving it into the controller?